requirements.txt

```diff
@@ -0,0 +1,5 @@
+numpy==1.16.0
```

Comment:
we want to support future versions too so use >= unless there's a good reason not to

Reply:
🎉